frontend/src/components/admin/AssignedStoryTranslationsTable.tsx

```diff
@@ -0,0 +1,176 @@
+    const newStoryTranslations = [...storyTranslations];
+    switch (field) {
+      case "title": {
+        setIsAscendingTitle(!isAscendingTitle);
+
+        newStoryTranslations.sort(titleSort);
+        break;
+      }
+      case "role": {
+        setIsAscendingRole(!isAscendingRole);
+
+        newStoryTranslations.sort(roleSort);
+        break;
+      }
+      case "language": {
+        setIsAscendingLanguage(!isAscendingLanguage);
+
+        newStoryTranslations.sort(languageSort);
+        break;
+      }
+      case "stage": {
+        setIsAscendingStage(!isAscendingStage);
+
+        newStoryTranslations.sort(stageSort);
+        break;
+      }
+      default: {
+        break;
+      }
+    }
+    setStoryTranslations(newStoryTranslations);
+  };
+
+  const tableBody = storyTranslations.map(
+    (translation: StoryTranslation, index: number) => (
+      <Tr
+        borderBottom={
+          index === storyTranslations.length - 1 ? "1em solid transparent" : ""
+        }
+        key={`${translation?.storyTranslationId}`}
+      >
+        <Td>{translation.title}</Td>
+        <Td>{getRole(translation)}</Td>
+        <Td>
+          <Badge
+            background={getLevelVariant(translation.level)}
+            marginBottom="3px"
+            marginTop="3px"
+          >{`${convertLanguageTitleCase(translation.language)} | Level ${
+            translation.level
+          }`}</Badge>
+        </Td>
+        <Td>{convertStageTitleCase(translation.stage)}</Td>
+        <Td>
+          <IconButton
+            aria-label={`Delete story translation for ${translation.title}`}
+            background="transparent"
+            icon={<Icon as={MdDelete} />}
+            width="fit-content"
+          />
+        </Td>
+      </Tr>
+    ),
+  );
+  return (
+    <Table
+      borderRadius="12px"
+      boxShadow="0px 0px 2px grey"
+      size="sm"
+      theme="gray"
+      variant="striped"
+      width="100%"
+    >
+      <Thead>
+        <Tr
+          borderTop="1em solid transparent"
+          borderBottom="0.5em solid transparent"
+        >
+          <Th cursor="pointer" onClick={() => sort("title")}>
+            {`BOOK TITLE ${isAscendingTitle ? "↑" : "↓"}`}{" "}
+          </Th>
+          <Th cursor="pointer" onClick={() => sort("role")}>{`ROLE ${
+            isAscendingRole ? "↑" : "↓"
+          }`}</Th>
+          <Th
+            cursor="pointer"
+            onClick={() => sort("language")}
+          >{`LANGUAGE & LEVEL ${isAscendingLanguage ? "↑" : "↓"}`}</Th>
+          <Th cursor="pointer" onClick={() => sort("stage")}>{`PROGRESS ${
+            isAscendingStage ? "↑" : "↓"
+          }`}</Th>
+          <Th>ACTION</Th>
+        </Tr>
+      </Thead>
+      <Tbody>{tableBody}</Tbody>
+      {/* <hr marginTop="-10px"/>  How do I add a horizontal line lol */}
+      <Button
+        variant="blueOutline"
+        size="secondary"
+        margin="5px 0px 15px 10px"
+        width="200px"
+        border="2px"
```

Comment:
why set the border style?

Reply:
The border was really thin so I added the additionally styling to get it to match with the figma better